posts/embracing_startup_agility_in_retail.md

```diff
@@ -0,0 +1,61 @@
+6 Ways to Embrace Startup Agility in Retail
+===========================================
+
+The retail industry has been squeezed on many fronts over the past decade, from Internet giants flexing their technology muscle (and willingness to not always turn a profit) to disrupt markets, to fast-moving startups who are unincumbered by big company process, and who are able to attract strong technology talent.
+
+As companies grow, it is an almost enevitable rite of passage that stricter processes are embedded, decision making can become more centralised, and increasing layers added to the heirarchy. While in some measure these things can help an organisation to scale, they can significantly reduce an organisations agility when it comes to embracing changing commercial landscapes.
+
+Established retailers can take conscious steps toward adopting and embracing startup levels of agility, ideally without the relentless late nights.
+
+We've observed 6 ways retailers can embrace startup agility:
+
+
+## 1. Empower and incentivise teams
+
+Particularly in large organisations, individuals and teams may not be fully empowered to deliver their program of work. Teams that are reliant on outside stakeholders for decisions or for additional support typically show a much lower level of performance than a team that is empowered and incentivised for end-to-end delivery.
```

Comment:
Playing the role of someone in a large organisation, I'd like to know a bit more about incentivising people for end-to-end delivery.